### PR TITLE
[feature](Cloud) Generate one suffix to distinguish different hdfs root path

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

File: cloud/src/meta-service/meta_service_resource.cpp [hunk 1]
```diff
@@ -387,6 +388,13 @@ static int add_hdfs_storage_vault(InstanceInfoPB& instance, Transaction* txn,
         msg = fmt::format("invalid prefix: ", *prefix);
         return -1;
     }
+    auto uuid_suffix = butil::GenerateGUID();
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_resource.cpp:391]: add a config to enable this optimization

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by ByteYue** [comment]
run buildall

**Review by platoneko — DISMISSED** [review]

**Comment by ByteYue** [comment]
run buildall

**Review by platoneko — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Pull request merged**